common/shlibs

```diff
@@ -800,6 +800,7 @@ libsvn_ra_serf-1.so.0 libsvn-1.8.1_1
 libsvn_fs_x-1.so.0 libsvn-1.9.0_1
 libsvn_swig_py-1.so.0 subversion-python-1.7.5_1
 libsvn_swig_perl-1.so.0 subversion-perl-1.7.5_1
+libjson-c.so.5 json-c-0.14.0_1
```

Comment:
this package needs to be revbumped, I think

Reply:
an [update of libjson-c](https://github.com/void-linux/void-packages/pull/52634) has already been merged so I had to drop a libjson-c commit to resolve a rebase conflict. Also, there was no revbump in _common/shlibs_ in the aformentioned PR. Are you sure this update needs the revbump?

Reply:
why are you adding that to common/shlibs? it already exists on line 884